setup-rules.md

```diff
@@ -58,3 +62,39 @@ Request must be submitted, reviewed, and merged before updating the
 superproject.
 
 [submodules]: https://www.git-scm.com/book/en/v2/Git-Tools-Submodules
+
+
+
+### How We Use Submodules
+
+Many of our products are dependent on other projects. We need two
+things when we build projects that make use of dependencies:
+
+1. Many of our builds require dependent projects to be in known
+specific locations so one project can trigger a build of a related
+project, and
+
+2. For support purposes, we need to be able to identify the specific
+sources used to build a complete project, even if the project includes
+other common components. This allows us to review actual files used
+for the build when defects or bugs are reported.
+
+That is exactly how we use super-projects and submodules. The
+super-project contains the list of all repositories that are to be
+used to build the overall project **and** where those projects are
+checked out relative to one another, thus satisfying #1 above.
+
+Since the super-project also contains specific commit hashes for each
+subproject used to perform the build, this satisfies #2 above.
```

Comment:
Thanks for highlighting this. That super projects point to specific commits was what I missed before reading your email.